tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -392,16 +386,15 @@
             "_scan": "https://ftmscan.com/token/0xeaF45B62d9d0Bdc1D763baF306af5eDD7C0d7e55",
             "chainId": 250,
             "decimals": 9,
-            "logoURI": "https://assets.spookyswap.finance/tokens/DEATH.png"
+            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/11994.png"
         },
         {
             "name": "The Eye of Truth",
             "symbol": "EYE",
             "address": "0x496e1693A7B162c4f0Cd6a1792679cC48EcbCC8d",
             "_scan": "https://ftmscan.com/token/0x496e1693A7B162c4f0Cd6a1792679cC48EcbCC8d",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/EYE.png"
+            "decimals": 18
```

Comment:
add logo

Reply:
The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it